### PR TITLE
NRS demo- custom component- google map

### PR DESCRIPTION
**Pull request by Josephalexantony-aot**
Created new Google Map Custom component for NRS Demo

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/customFormioComponents/Google-Map/Google_Map.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            location: '',
+            value: props.value,
+            gmapurl: process.env.REACT_APP_GOOGLE_MAP_URL
+        };
+    }
+    componentDidMount() {
+        if (navigator.geolocation) {
+            //TODO set the position of the map with value here
+            navigator.geolocation.getCurrentPosition(position => {
+                console.log("current Position", position);
```
Comment (abhilash-aot) [inline forms-flow-web/src/customFormioComponents/Google-Map/Google_Map.js:28]: Remove consoles

File: forms-flow-web/src/customFormioComponents/Google-Map/Google_Map.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    render() {
+        let { coordinates, placeName } = this.state.value ? this.state.value : {
+            placeName: '',
+            coordinates: {
+                lat: 0,
+                lng: 0
+            }
+        };
+        console.log("coordinates", coordinates, "placeName", placeName);
+        const AsyncMap = withScriptjs(
```
Comment (abhilash-aot) [inline forms-flow-web/src/customFormioComponents/Google-Map/Google_Map.js:116]: Console remove here too

**Review by abhilash-aot — APPROVED** [review]
Remove consoles and merge

File: forms-flow-web/src/customFormioComponents/Google-Map/Google_Map.js (hunk 1 above)
Comment (Josephalexantony-aot) [reply forms-flow-web/src/customFormioComponents/Google-Map/Google_Map.js:28]: sure

**Pull request merged**